JSONStorage.swift

```diff
@@ -160,27 +160,27 @@ extension JSONStorage {
             return Observable.just(self.memoryCache)
         }
         
-        return Observable.create({ (observer) -> Disposable in
+        return Observable.create({ [weak self] (observer) -> Disposable in
             
-            guard let storeUrl = self.storeUrl else {
+            guard let storeUrl = self?.storeUrl else {
```

Comment:
I would preffer guard let `self` here

Reply:
I'll rewrite that to perform continuous updates for that reactive signal 😄

Reply:
@piotrbernad Done that, please check it now